### ISSUE TITLE
QCOW2: can't update from 4.8 to 4.9

### ISSUE THREAD

**Issue by SAOPP** [home-assistant/operating-system#738]
**HassOS release with the issue:**

HassOS 4.8

**Supervisor logs:**

```
20-06-11 11:02:58 INFO (SyncWorker_8) [supervisor.docker.interface] Cleanup images: ['homeassistant/qemux86-64-homeassistant:0.110.4']
20-06-11 11:25:43 INFO (MainThread) [supervisor.hassos] Fetch OTA update from https://github.com/home-assistant/operating-system/releases/download/4.9/hassos_ova-4.9.raucb
20-06-11 11:35:44 WARNING (MainThread) [supervisor.hassos] Can't fetch versions from https://github.com/home-assistant/operating-system/releases/download/4.9/hassos_ova-4.9.raucb: 
20-06-11 11:35:44 ERROR (MainThread) [asyncio] Task exception was never retrieved
future: <Task finished coro=<HassOS.update() done, defined at /usr/src/supervisor/supervisor/hassos.py:125> exception=HassOSUpdateError()>
Traceback (most recent call last):
  File "/usr/src/supervisor/supervisor/hassos.py", line 136, in update
    int_ota = await self._download_raucb(version)
  File "/usr/src/supervisor/supervisor/hassos.py", line 88, in _download_raucb
    raise HassOSUpdateError()
supervisor.exceptions.HassOSUpdateError
```
**Journal logs:**

**Kernel logs:**

**Description of problem:**
Guys, I'm trying to update my current instance from 4.8 to 4.9 (qcow2) and I get the same error twice.

**Comment by SAOPP** [comment]
I see that now 4.10 is the latest version, probably my issue maybe due to the fact that 4.9 is no longer available, and an update on 4.10 has not yet appeared for me. I will continue to observe.

**Comment by SAOPP** [comment]
also if I stay on update page, I can see this after some time:

![image](https://user-images.githubusercontent.com/5647009/84384270-669b9c00-abf6-11ea-8338-8ea233557140.png)
